app/src/main/java/com/android/kotlinlearning/Rastaurant1/CastomAdapter1.kt

```diff
@@ -0,0 +1,48 @@
+package com.android.kotlinlearning.Rastaurant1
```

Comment:
Please check the spelling while you write, CastomAdapter.

follow package naming convention.